src/update/index.js

```diff
@@ -118,6 +55,10 @@ function getFileUpdater(root, config, progress) {
      * @param {Transformer} transform
      */
     return (relPath, transform) => {
+        // check if path matches any of the excludes patterns
+        const excludes = config.template_pull?.exclude_file_paths ?? [];
+        if (excludes.some(exclude => new RegExp(exclude).test(relPath))) return;
```

Comment:
Feel like this could lead to some confusion as the option is named `template_pull`. Maybe we could add an exclude option to `UpdateOptions`? Then we could pass the excluded files from `pull` instead

Reply:
Thanks. I converted to just `config.exclude_file_paths` as it should exclude running the update on those files as well